### PR TITLE
feat(components/indicators): help inline visual test

### PR DESCRIPTION
**Pull request by Blackbaud-SandhyaRajasabeson**
#[AB2195464](https://blackbaud.visualstudio.com/Products/_boards/board/t/SKY%20UX%20Program/Stories/?workitem=2195464)

### CODE REVIEW

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/1160/storybook/)

Component Storybooks:

- [a11y-storybook](https://blackbaud.github.io/skyux-pr-preview/1160/storybooks/a11y-storybook/)
- [ag-grid-storybook](https://blackbaud.github.io/skyux-pr-preview/1160/storybooks/ag-grid-storybook/)
- [core-storybook](https://blackbaud.github.io/skyux-pr-preview/1160/storybooks/core-storybook/)
- [indicators-storybook](https://blackbaud.github.io/skyux-pr-preview/1160/storybooks/indicators-storybook/)
- [layout-storybook](https://blackbaud.github.io/skyux-pr-preview/1160/storybooks/layout-storybook/)
- [modals-storybook](https://blackbaud.github.io/skyux-pr-preview/1160/storybooks/modals-storybook/)
- [pages-storybook](https://blackbaud.github.io/skyux-pr-preview/1160/storybooks/pages-storybook/)
- [popovers-storybook](https://blackbaud.github.io/skyux-pr-preview/1160/storybooks/popovers-storybook/)
- [split-view-storybook](https://blackbaud.github.io/skyux-pr-preview/1160/storybooks/split-view-storybook/)
- [storybook](https://blackbaud.github.io/skyux-pr-preview/1160/storybooks/storybook/)
- [tabs-storybook](https://blackbaud.github.io/skyux-pr-preview/1160/storybooks/tabs-storybook/)
- [theme-storybook](https://blackbaud.github.io/skyux-pr-preview/1160/storybooks/theme-storybook/)
- [tiles-storybook](https://blackbaud.github.io/skyux-pr-preview/1160/storybooks/tiles-storybook/)

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/1160/code-examples/)
- [dep-graph](https://blackbaud.github.io/skyux-pr-preview/1160/dep-graph/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/1160/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/1160/playground/)

File: apps/e2e/indicators-storybook/src/app/help-inline/help-inline.component.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<h2>
+  Title with inline help
+  <sky-help-inline class="sky-control-help"></sky-help-inline>
```
Comment (Blackbaud-SteveBrush) [inline apps/e2e/indicators-storybook/src/app/help-inline/help-inline.component.html]: I can't tell from the screenshot, but is the fact that this is on a new line adding more padding between the heading text and help-inline button than desired?

Comment (Blackbaud-SandhyaRajasabeson) [reply apps/e2e/indicators-storybook/src/app/help-inline/help-inline.component.html]: moving `sky-help-inline` to be actually inline with the heading text fixed this issue

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Comment by blackbaud-sky-build-user** [comment]
Cherry-pick successful! :cherries: :tada: See null

**Pull request merged**